app/services/jokes.js

```diff
@@ -0,0 +1,21 @@
+const axios = require('axios');
+
+const errors = require('../errors');
+const config = require('../../config');
+
+const { baseUrl } = config.common.geekJokesApi;
+
+exports.getRandom = () =>
```

Comment:
We should use a more descriptive name. If there are other random functions that are called `getRandom`, we will have to give aliases to them when we require them.